lib/Transforms/AddOuterParallelLoop.cpp

```diff
@@ -31,7 +31,7 @@ struct AddOuterParallelLoopPass
     auto func = getOperation();
     if (func.getBody().empty())
       return;
-    std::vector<llvm::SmallVector<Operation *, 4>> groupedOps;
+    llvm::SmallVector<llvm::SmallVector<Operation *, 4>, 4> groupedOps;
```

Comment:
Is "4" here specifies value? If So, is there specific reason to initialize with 4?

Reply:
it's a common practice in llvm to specify it as 4 or 8 according to the using case.